src/compiler/binder.ts

```diff
@@ -1521,6 +1565,17 @@ namespace ts {
                             }
                             completeNode();
                         }
+                        else if(isLogicalAssignmentOperator(operator)) {

```

Comment:
```suggestion
                        else if (isLogicalAssignmentOperator(operator)) {
```

Reply:
Code style and reuse have not been adjusted yet.🤦🏻‍♂️Binder spend all my energy。

Reply:
Sorry, didn't mean to jump at you with style nits. 😅

By the way, you've got a lot of time to work on this since `master` is only open to new features starting on April 24th: https://github.com/microsoft/TypeScript/issues/37198

That said, feel free to keep iterating. It's been really fun for me to watch the progress here.